src/main/ts/components/Editor.tsx

```diff
@@ -215,6 +215,13 @@ export class Editor extends React.Component<IAllProps> {
         if (this.props.init && isFunction(this.props.init.setup)) {
           this.props.init.setup(editor);
         }
+      },
+      init_instance_callback: (editor) => {
```

Comment:
This method is good but you need to check if the user defined `init_instance_callback` themselves and call it if they did.